File: evidence/generated-systems/angular/hr-on-angular/src/app/models/PayrollCalendar.js

// Define collection and schema for PayrollCalendar
export interface PayrollCalendar {
    name:
	type : string
    country:
	type : string
    Organization:
	type : Schema.Types.ObjectId
    PayrollRuns:
 	type : [{ type: Schema.Types.ObjectId, ref: 'PayrollRun' }]
    Employees:
 	type : [{ type: Schema.Types.ObjectId, ref: 'Employee' }]
    PayFrequency:
 	type : String
#
    collection: 'payrollCalendars'
}

File: evidence/generated-systems/rubyonrails/hr-on-rails/hronrails/app/models/payrollcalendar.rb
class PayrollCalendar < ApplicationRecord
  enum PayFrequency: [:Weekly, :Biweekly, :Semimonthly, :Monthly, :Quarterly]


  has_many :Organization, class_name: 'Organization'
  has_many :PayrollRuns, class_name: 'PayrollRun'
  has_many :Employees, class_name: 'Employee'

end
